Creating-rosetta-terminal.md

```diff
@@ -0,0 +1,17 @@
+## How to create a Rosetta 2 terminal on Mac M1 computers
```

Comment:
Can we put these instructions in the mac installation instructions? Since they're so short, it seems unnecessary to have a whole separate page for them

Reply:
Yeah, guess the most reasonable place would be in step 1 in "Macs with M1 chips"?